codex/slots/proofs/backendfactory.nim

```diff
@@ -0,0 +1,88 @@
+import os
+import strutils
+import pkg/chronos
+import pkg/chronicles
+import pkg/questionable
+import pkg/confutils/defs
+import pkg/stew/io2
+import pkg/ethers
+
+import ../../conf
+import ./backends
+import ./backendutils
+
+type
```

Comment:
This is nitpicking a little bit, but you really don't need this placeholder, just attach the factory to the config object?

Reply:
All right now I see what you mean. Yes this is better. 👍